packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/location.tsp

```diff
@@ -0,0 +1,179 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@resource("locationTrackedResources")
+@parentResource(ResourceGroupLocationResource)
+model LocationTrackedResource is TrackedResource<LocationTrackedResourceProperties> {
```

Comment:
does it matter more that it's tracked, or that it's location tracked. I'm not seeing anything really "location" about it besides the name. I feel it woujld be better to call it `models/resources/tracked` and the scenario names to have `TrackedResource` as the name, since that corresponds directly with the model name from the ARM library

Reply:
> does it matter more that it's tracked, or that it's location tracked

It's basically to test `@parentResource(ResourceGroupLocationResource)`, not tracked resource. Whether it's tracked doesn't matter, it could also be `ProxyResource`.

Reason to call it `LocationTrackedResource` is to avoid being to generic, e.g. `LocationResource`, with segment `locations`. 
Yeah, the name is still a bit confusing. @v-hongli1 Could you try if we could rename the model to `RgLocationResource`(since `ResourceGroupLocationResource` is already occupied)?